content/2023-04-24-t8code_version120_released.md

```diff
@@ -0,0 +1,17 @@
+
+
+Title: t8code-1.2.0 released Date: 2023-04-24 17:00 Category: News Authors: Johannes Holke, Sandro Elsweijer, Ioannis Lilikakis
+
+We have released version 1.2.0 of t8code!
+
+New features include:
+
+- A new cmesh reader for VTK files.
```

Comment:
Only this bullet point ends with a period. Decide which style to use for all bullet points.

Reply:
done